trace_decoder/src/decoding.rs

```diff
@@ -25,33 +25,48 @@ use crate::{
     utils::{hash, update_val_if_some},
 };
 
+/// Stores the result of parsing tries. Returns a `TraceParsingError` in case of
+/// an error.
```

Comment:
Neat trick: if the type/trait is in scope, surrounding its name with `[`/`]` will make rustdoc automatically create a link to it. If it's not in scope it will just not add a link. So here, `[TraceParsingError]` would generate a direct link to the type def.